activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryAdminProcessServiceTasksIT.java

```diff
@@ -0,0 +1,222 @@
+public class QueryAdminProcessServiceTasksIT {
+
+    private static final String PROC_URL = "/admin/v1/process-instances";
+
+    private static final ParameterizedTypeReference<PagedModel<BPMNActivity>> PAGED_TASKS_RESPONSE_TYPE = new ParameterizedTypeReference<PagedModel<BPMNActivity>>() {
+    };
+
+    private static final ParameterizedTypeReference<BPMNActivity> SINGLE_TASK_RESPONSE_TYPE = new ParameterizedTypeReference<BPMNActivity>() {
+    };
+
+    @Autowired
+    private KeycloakTokenProducer keycloakTokenProducer;
+
+    @Autowired
+    private ProcessDefinitionRepository processDefinitionRepository;
+
+    @Autowired
+    private ProcessModelRepository processModelRepository;
+
+    @Autowired
+    private ProcessInstanceRepository processInstanceRepository;
+
+    @Autowired
+    private BPMNActivityRepository bpmnActivityRepository;
+
+    @Autowired
+    private BPMNSequenceFlowRepository bpmnSequenceFlowRepository;
+
+    @Autowired
+    private MyProducer producer;
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
+
+    private String processDefinitionId = UUID.randomUUID().toString();
+
+
+    private EventsAggregator eventsAggregator;
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        keycloakTokenProducer.setKeycloakTestUser("hradmin");
+
+        eventsAggregator = new EventsAggregator(producer);
+
+        //given
+        ProcessDefinitionImpl firstProcessDefinition = new ProcessDefinitionImpl();
+        firstProcessDefinition.setId(processDefinitionId);
+        firstProcessDefinition.setKey("mySimpleProcess");
+        firstProcessDefinition.setName("My Simple Process");
+
+        CloudProcessDeployedEventImpl firstProcessDeployedEvent = new CloudProcessDeployedEventImpl(firstProcessDefinition);
+        firstProcessDeployedEvent.setProcessModelContent(new String(Files.readAllBytes(Paths.get("src/test/resources/parse-for-test/SimpleProcess.bpmn20.xml")),
+                                                                    StandardCharsets.UTF_8));
+
+        producer.send(firstProcessDeployedEvent);
+    }
+
+    @AfterEach
+    public void tearDown() {
+        processModelRepository.deleteAll();
+        processDefinitionRepository.deleteAll();
+        processInstanceRepository.deleteAll();
+        bpmnActivityRepository.deleteAll();
+        bpmnSequenceFlowRepository.deleteAll();
+    }
+
+    @Test
+    public void shouldGetProcessInstanceServiceTasks() throws InterruptedException {
+        //given
+        ProcessInstanceImpl process = startSimpleProcessInstance();
+
+        //when
+        eventsAggregator.sendAll();
+
+        //then
+        await().untilAsserted(() -> {
+            assertThat(bpmnActivityRepository.findByProcessInstanceId(process.getId())).hasSize(2);
+            assertThat(bpmnSequenceFlowRepository.findByProcessInstanceId(process.getId())).hasSize(1);
+        });
+
+        await().atMost(Durations.ONE_MINUTE).untilAsserted(() -> {
```

Comment:
Unless `.atMost(Durations.ONE_MINUTE)` was added because the test was failing with the default configuration of `await`, it's better to keep defaults (don't specify `atMost`).

Reply:
fixed